toolkit/imageconfigs/packagelists/core-packages-image.json

```diff
@@ -6,6 +6,11 @@
         "cronie-anacron",
         "logrotate",
         "core-packages-base-image",
+        "dracut-hostonly",
```

Comment:
core-packages-image makes it into both the general core images as well as the marketplace. I think marketplace wants just hostonly, vrf, hyperv. The core images want everything. We'll need to compose these package lists slightly differently to achieve that.